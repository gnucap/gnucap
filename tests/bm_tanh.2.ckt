'
V1  1  0  DC   1.
E1  2  0  1  0  TANH  gain= 100.  limit= 0.01
.print dc v nodes
.dc v1 -.00000001 .00000001 .000000001
.dc v1 -.0000001 .0000001 .00000001
.dc v1 -.000001 .000001 .0000001
.dc v1 -.00001 .00001 .000001
.dc v1 -.0001 .0001 .00001
.dc v1 -.001 .001 .0001
.dc v1 -.01 .01 .001
.dc v1 -.1 .1 .01
.dc v1 -1 1 .1
.dc v1 -10 10 1
.dc v1 -100 100 10
.dc v1 -1000 1000 100
.dc v1 -10k 10k 1k
.dc v1 -100k 100k 10k
.dc v1 -1000k 1000k 100k
.dc v1 -10000k 10000k 1000k
*>.status notime
.end
