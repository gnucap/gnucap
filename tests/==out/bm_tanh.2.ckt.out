' 
#           v(1)       v(2)      
-10.n      -10.n      -1.u       
-9.n       -9.n       -900.n     
-8.n       -8.n       -800.n     
-7.n       -7.n       -700.n     
-6.n       -6.n       -600.n     
-5.n       -5.n       -500.n     
-4.n       -4.n       -400.n     
-3.n       -3.n       -300.n     
-2.n       -2.n       -200.n     
-1.n       -1.n       -100.n     
 0.         0.         0.        
 1.n        1.n        100.n     
 2.n        2.n        200.n     
 3.n        3.n        300.n     
 4.n        4.n        400.n     
 5.n        5.n        500.n     
 6.n        6.n        600.n     
 7.n        7.n        700.n     
 8.n        8.n        800.n     
 9.n        9.n        900.n     
 10.n       10.n       1.u       
#           v(1)       v(2)      
-100.n     -100.n     -10.u      
-90.n      -90.n      -9.u       
-80.n      -80.n      -8.u       
-70.n      -70.n      -7.u       
-60.n      -60.n      -6.u       
-50.n      -50.n      -5.u       
-40.n      -40.n      -4.u       
-30.n      -30.n      -3.u       
-20.n      -20.n      -2.u       
-10.n      -10.n      -1.u       
 0.         0.         0.        
 10.n       10.n       1.u       
 20.n       20.n       2.u       
 30.n       30.n       3.u       
 40.n       40.n       4.u       
 50.n       50.n       5.u       
 60.n       60.n       6.u       
 70.n       70.n       7.u       
 80.n       80.n       8.u       
 90.n       90.n       9.u       
 100.n      100.n      10.u      
#           v(1)       v(2)      
-1.u       -1.u       -99.997u   
-900.n     -900.n     -89.998u   
-800.n     -800.n     -79.998u   
-700.n     -700.n     -69.999u   
-600.n     -600.n     -59.999u   
-500.n     -500.n     -50.u      
-400.n     -400.n     -40.u      
-300.n     -300.n     -30.u      
-200.n     -200.n     -20.u      
-100.n     -100.n     -10.u      
 0.         0.         0.        
 100.n      100.n      10.u      
 200.n      200.n      20.u      
 300.n      300.n      30.u      
 400.n      400.n      40.u      
 500.n      500.n      50.u      
 600.n      600.n      59.999u   
 700.n      700.n      69.999u   
 800.n      800.n      79.998u   
 900.n      900.n      89.998u   
 1.u        1.u        99.997u   
#           v(1)       v(2)      
-10.u      -10.u      -996.68u   
-9.u       -9.u       -897.58u   
-8.u       -8.u       -798.3u    
-7.u       -7.u       -698.86u   
-6.u       -6.u       -599.28u   
-5.u       -5.u       -499.58u   
-4.u       -4.u       -399.79u   
-3.u       -3.u       -299.91u   
-2.u       -2.u       -199.97u   
-1.u       -1.u       -99.997u   
 0.         0.         0.        
 1.u        1.u        99.997u   
 2.u        2.u        199.97u   
 3.u        3.u        299.91u   
 4.u        4.u        399.79u   
 5.u        5.u        499.58u   
 6.u        6.u        599.28u   
 7.u        7.u        698.86u   
 8.u        8.u        798.3u    
 9.u        9.u        897.58u   
 10.u       10.u       996.68u   
#           v(1)       v(2)      
-100.u     -100.u     -0.0076159 
-90.u      -90.u      -0.007163  
-80.u      -80.u      -0.0066404 
-70.u      -70.u      -0.0060437 
-60.u      -60.u      -0.0053705 
-50.u      -50.u      -0.0046212 
-40.u      -40.u      -0.0037995 
-30.u      -30.u      -0.0029131 
-20.u      -20.u      -0.0019738 
-10.u      -10.u      -996.68u   
 0.         0.         0.        
 10.u       10.u       996.68u   
 20.u       20.u       0.0019738 
 30.u       30.u       0.0029131 
 40.u       40.u       0.0037995 
 50.u       50.u       0.0046212 
 60.u       60.u       0.0053705 
 70.u       70.u       0.0060437 
 80.u       80.u       0.0066404 
 90.u       90.u       0.007163  
 100.u      100.u      0.0076159 
#           v(1)       v(2)      
-0.001     -0.001     -0.01      
-900.u     -900.u     -0.01      
-800.u     -800.u     -0.01      
-700.u     -700.u     -0.01      
-600.u     -600.u     -0.0099999 
-500.u     -500.u     -0.0099991 
-400.u     -400.u     -0.0099933 
-300.u     -300.u     -0.0099505 
-200.u     -200.u     -0.0096403 
-100.u     -100.u     -0.0076159 
 0.         0.         0.        
 100.u      100.u      0.0076159 
 200.u      200.u      0.0096403 
 300.u      300.u      0.0099505 
 400.u      400.u      0.0099933 
 500.u      500.u      0.0099991 
 600.u      600.u      0.0099999 
 700.u      700.u      0.01      
 800.u      800.u      0.01      
 900.u      900.u      0.01      
 0.001      0.001      0.01      
#           v(1)       v(2)      
-0.01      -0.01      -0.01      
-0.009     -0.009     -0.01      
-0.008     -0.008     -0.01      
-0.007     -0.007     -0.01      
-0.006     -0.006     -0.01      
-0.005     -0.005     -0.01      
-0.004     -0.004     -0.01      
-0.003     -0.003     -0.01      
-0.002     -0.002     -0.01      
-0.001     -0.001     -0.01      
 0.         0.         0.        
 0.001      0.001      0.01      
 0.002      0.002      0.01      
 0.003      0.003      0.01      
 0.004      0.004      0.01      
 0.005      0.005      0.01      
 0.006      0.006      0.01      
 0.007      0.007      0.01      
 0.008      0.008      0.01      
 0.009      0.009      0.01      
 0.01       0.01       0.01      
#           v(1)       v(2)      
-0.1       -0.1       -0.01      
-0.09      -0.09      -0.01      
-0.08      -0.08      -0.01      
-0.07      -0.07      -0.01      
-0.06      -0.06      -0.01      
-0.05      -0.05      -0.01      
-0.04      -0.04      -0.01      
-0.03      -0.03      -0.01      
-0.02      -0.02      -0.01      
-0.01      -0.01      -0.01      
 0.         0.         0.        
 0.01       0.01       0.01      
 0.02       0.02       0.01      
 0.03       0.03       0.01      
 0.04       0.04       0.01      
 0.05       0.05       0.01      
 0.06       0.06       0.01      
 0.07       0.07       0.01      
 0.08       0.08       0.01      
 0.09       0.09       0.01      
 0.1        0.1        0.01      
#           v(1)       v(2)      
-1.        -1.        -0.01      
-0.9       -0.9       -0.01      
-0.8       -0.8       -0.01      
-0.7       -0.7       -0.01      
-0.6       -0.6       -0.01      
-0.5       -0.5       -0.01      
-0.4       -0.4       -0.01      
-0.3       -0.3       -0.01      
-0.2       -0.2       -0.01      
-0.1       -0.1       -0.01      
 0.         0.         0.        
 0.1        0.1        0.01      
 0.2        0.2        0.01      
 0.3        0.3        0.01      
 0.4        0.4        0.01      
 0.5        0.5        0.01      
 0.6        0.6        0.01      
 0.7        0.7        0.01      
 0.8        0.8        0.01      
 0.9        0.9        0.01      
 1.         1.         0.01      
#           v(1)       v(2)      
-10.       -10.       -0.01      
-9.        -9.        -0.01      
-8.        -8.        -0.01      
-7.        -7.        -0.01      
-6.        -6.        -0.01      
-5.        -5.        -0.01      
-4.        -4.        -0.01      
-3.        -3.        -0.01      
-2.        -2.        -0.01      
-1.        -1.        -0.01      
 0.         0.         0.        
 1.         1.         0.01      
 2.         2.         0.01      
 3.         3.         0.01      
 4.         4.         0.01      
 5.         5.         0.01      
 6.         6.         0.01      
 7.         7.         0.01      
 8.         8.         0.01      
 9.         9.         0.01      
 10.        10.        0.01      
#           v(1)       v(2)      
-100.      -100.      -0.01      
-90.       -90.       -0.01      
-80.       -80.       -0.01      
-70.       -70.       -0.01      
-60.       -60.       -0.01      
-50.       -50.       -0.01      
-40.       -40.       -0.01      
-30.       -30.       -0.01      
-20.       -20.       -0.01      
-10.       -10.       -0.01      
 0.         0.         0.        
 10.        10.        0.01      
 20.        20.        0.01      
 30.        30.        0.01      
 40.        40.        0.01      
 50.        50.        0.01      
 60.        60.        0.01      
 70.        70.        0.01      
 80.        80.        0.01      
 90.        90.        0.01      
 100.       100.       0.01      
#           v(1)       v(2)      
-1.K       -1.K       -0.01      
-900.      -900.      -0.01      
-800.      -800.      -0.01      
-700.      -700.      -0.01      
-600.      -600.      -0.01      
-500.      -500.      -0.01      
-400.      -400.      -0.01      
-300.      -300.      -0.01      
-200.      -200.      -0.01      
-100.      -100.      -0.01      
 0.         0.         0.        
 100.       100.       0.01      
 200.       200.       0.01      
 300.       300.       0.01      
 400.       400.       0.01      
 500.       500.       0.01      
 600.       600.       0.01      
 700.       700.       0.01      
 800.       800.       0.01      
 900.       900.       0.01      
 1.K        1.K        0.01      
#           v(1)       v(2)      
-10.K      -10.K      -0.01      
-9.K       -9.K       -0.01      
-8.K       -8.K       -0.01      
-7.K       -7.K       -0.01      
-6.K       -6.K       -0.01      
-5.K       -5.K       -0.01      
-4.K       -4.K       -0.01      
-3.K       -3.K       -0.01      
-2.K       -2.K       -0.01      
-1.K       -1.K       -0.01      
 0.         0.         0.        
 1.K        1.K        0.01      
 2.K        2.K        0.01      
 3.K        3.K        0.01      
 4.K        4.K        0.01      
 5.K        5.K        0.01      
 6.K        6.K        0.01      
 7.K        7.K        0.01      
 8.K        8.K        0.01      
 9.K        9.K        0.01      
 10.K       10.K       0.01      
#           v(1)       v(2)      
-100.K     -100.K     -0.01      
-90.K      -90.K      -0.01      
-80.K      -80.K      -0.01      
-70.K      -70.K      -0.01      
-60.K      -60.K      -0.01      
-50.K      -50.K      -0.01      
-40.K      -40.K      -0.01      
-30.K      -30.K      -0.01      
-20.K      -20.K      -0.01      
-10.K      -10.K      -0.01      
 0.         0.         0.        
 10.K       10.K       0.01      
 20.K       20.K       0.01      
 30.K       30.K       0.01      
 40.K       40.K       0.01      
 50.K       50.K       0.01      
 60.K       60.K       0.01      
 70.K       70.K       0.01      
 80.K       80.K       0.01      
 90.K       90.K       0.01      
 100.K      100.K      0.01      
#           v(1)       v(2)      
-1.Meg     -1.Meg     -0.01      
-900.K     -900.K     -0.01      
-800.K     -800.K     -0.01      
-700.K     -700.K     -0.01      
-600.K     -600.K     -0.01      
-500.K     -500.K     -0.01      
-400.K     -400.K     -0.01      
-300.K     -300.K     -0.01      
-200.K     -200.K     -0.01      
-100.K     -100.K     -0.01      
 0.         0.         0.        
 100.K      100.K      0.01      
 200.K      200.K      0.01      
 300.K      300.K      0.01      
 400.K      400.K      0.01      
 500.K      500.K      0.01      
 600.K      600.K      0.01      
 700.K      700.K      0.01      
 800.K      800.K      0.01      
 900.K      900.K      0.01      
 1.Meg      1.Meg      0.01      
#           v(1)       v(2)      
-10.Meg    -10.Meg    -0.01      
-9.Meg     -9.Meg     -0.01      
-8.Meg     -8.Meg     -0.01      
-7.Meg     -7.Meg     -0.01      
-6.Meg     -6.Meg     -0.01      
-5.Meg     -5.Meg     -0.01      
-4.Meg     -4.Meg     -0.01      
-3.Meg     -3.Meg     -0.01      
-2.Meg     -2.Meg     -0.01      
-1.Meg     -1.Meg     -0.01      
 0.         0.         0.        
 1.Meg      1.Meg      0.01      
 2.Meg      2.Meg      0.01      
 3.Meg      3.Meg      0.01      
 4.Meg      4.Meg      0.01      
 5.Meg      5.Meg      0.01      
 6.Meg      6.Meg      0.01      
 7.Meg      7.Meg      0.01      
 8.Meg      8.Meg      0.01      
 9.Meg      9.Meg      0.01      
 10.Meg     10.Meg     0.01      
Gnucap   System status
iterations: op=0, dc=84, tran=0, fourier=0, total=1344
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=2, subckt=0, model=0, total=2
dctran density=100.0%, ac density=100.0%
